dataservice/api/read_group/models.py

```diff
@@ -27,9 +27,14 @@ class ReadGroup(db.Model, Base):
     quality_scale = db.Column(db.Text(),
                               doc='The scale used to encode quality scores')
 
-    genomic_files = db.relationship('GenomicFile',
-                                    secondary='read_group_genomic_file',
-                                    backref=db.backref('read_groups'))
+    genomic_files = association_proxy(
```

Comment:
Is this just for convenience?

Reply:
Yea its for testing mostly so that I can directly access the genomic_files of a read_group